src/test/java/com/puppycrawl/tools/checkstyle/meta/MetadataGeneratorUtilTest.java

```diff
@@ -31,24 +31,38 @@
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
+import org.itsallcode.io.Capturable;
+import org.itsallcode.junit.sysextensions.SystemOutGuard;
+import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
 
+import com.puppycrawl.tools.checkstyle.AbstractModuleTestSupport;
 import com.puppycrawl.tools.checkstyle.internal.utils.CheckUtil;
 
-public final class MetadataGeneratorUtilTest {
+@ExtendWith(SystemOutGuard.class)
+public final class MetadataGeneratorUtilTest extends AbstractModuleTestSupport {
```

Comment:
It is not a module but still extended for using `getCheckMessage(..)` otherwise, I had to hardcode it in English.